README.md

```diff
@@ -1 +1,54 @@
-pretend this readme is as interesting as the website
+![Logo](https://pbs.twimg.com/profile_images/1373353833330475008/rKWjiXET_400x400.png)
+
+
+# Bit Project Website
+
+This GitHub repo is the home 🏠 of the Bit Project website. 
+Here at the Bit Project We equip students with technical 
+superpowers by providing cutting-edge, open source learning 
+experiences to prepare students for careers in tech. 
+
+
+## Serverless Camp (A Bit Project Program) Sponsors
+
+ - ⭐️ [Microsoft](https://www.microsoft.com/en-us/)
+ - ⭐️ [Courier](https://www.courier.com/)
+
+
+
+## Projects From Former Bit Project Students
+
+Too see exactly the kind of work we do at Bit Project, 
+feel free to check out some student projects from 
+
+- Grace built [EcoMaps](https://www.bitproject.org/projects/reducing-carbon-emissions) and reduced carbon emissions 🌱
+- Jakub [analyzed vehicle telemetry data](https://www.bitproject.org/projects/vehicle-telemetry-sigma-link) in real-time 🚗
+- Kaley developed [TimeBlocker](https://www.bitproject.org/projects/saving-time-with-timeblocker) and Boosted Productivity 📚
+- Fifi [saved money shopping on Amazon](https://www.bitproject.org/projects/saving-money-with-serverless) with Serverless 🛒
+
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://github.com/shreythecray)
```

Comment:
Can you make all of these formatted as:
💙 Shreya Gupta ([@shreythecray](https://github.com/shreythecray)), Founder (Operations) 

Make sure to replace what's in the parenthesis with their github username of course